client/src/browser-table.html

```diff
@@ -269,7 +273,7 @@ <h1 label>Browser support</h1>
         </a>
       </div>
       <div class="row">
-        <a href="https://html.spec.whatwg.org/multipage/webappapis.html#integration-with-the-javascript-module-system" target="__blank" rel="noopener">
+        <a spec href="https://html.spec.whatwg.org/multipag/webappapis.html#integration-with-the-javascript-module-system" target="_blank" rel="noopener">
```

Comment:
The `multipage` was correct. It results in a 404 now 😢

Reply:
My bad!